tests/integ/test_transformer.py

```diff
@@ -419,8 +420,12 @@ def test_transform_tf_kms_network_isolation(
 
         with open(os.path.join(tmpdir, "tf-batch-output", "data.csv.out")) as f:
             result = json.load(f)
-            assert len(result["predictions"][0]["probabilities"]) == 10
-            assert result["predictions"][0]["classes"] >= 1
+            prediction_0 = result["predictions"][0]
+            if type(prediction_0) is dict:
+                assert len(result["predictions"][0]["probabilities"]) == 10
+                assert result["predictions"][0]["classes"] >= 1
+            else:
+                assert len(result["predictions"][0]) == 10
```

Comment:
This change was needed because the output produced by TFS, for the model trained using the TF 2.6 training script, for batches of inputs now seems to be a "list of lists" rather than a "list of dictionaries".